datasets/us/nc/med_exclusions/us_nc_med_exclusions.yaml

```
@@ -0,0 +1,33 @@
+title: US North Carolina Excluded Providers List
+entry_point: crawler.py
+prefix: us-mednc
+coverage:
+  frequency: daily
+  start: "2024-09-09"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of providers who have been terminated for cause from North Carolina's Medicaid and other Health Programs.
+description: |
+  NC Medicaid can bar providers from participaing in Medicare and/or other Health Programs.
+  > An excluded provider is not entitled to bill or be reimbursed for medical services
```

I think this will need a line break
```suggestion
  NC Medicaid can bar providers from participaing in Medicare and/or other Health Programs.

  > An excluded provider is not entitled to bill or be reimbursed for medical services
```